tools/containers/_common/pc-container-json-rpc.py

```
@@ -0,0 +1,109 @@
+    "kicad-cli": "/usr/bin/kicad-cli",
+    "cat": "/usr/bin/cat",
+    "ls": "/usr/bin/ls",
+    # Add more allowed commands with their full paths
+}
+
+
+@jsonrpc.method("execute")
+def handle_execute_command(command: t.List[str],
+                           stdin: str = None,
+                           cwd: str = None,
+                           input_files: t.Dict[str, str] = {},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)